src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -413,49 +436,68 @@ g.test('it_is_invalid_to_set_resolve_target_if_color_attachment_is_non_multisamp
     };
 
     t.tryRenderPass(false, descriptor);
-  }
-);
+  });
 
-g.test('check_the_use_of_multisampled_textures_as_color_attachments').fn(async t => {
-  const colorTexture = t.createTexture({ sampleCount: 1 });
-  const multisampledColorTexture = t.createTexture({ sampleCount: 4 });
+g.test('color_attachment_multisampled_usage')
+  .desc(
+    `
+  Test the usages of multisampled textures for color attachments.
+    - Succeed if using a multisampled color attachment without setting a resolve target.
+    - Fail if using multiple color attachments with different sample counts.
+  `
+  )
+  .fn(async t => {
+    const colorTexture = t.createTexture({ sampleCount: 1 });
+    const multisampledColorTexture = t.createTexture({ sampleCount: 4 });
 
-  {
-    // It is allowed to use a multisampled color attachment without setting resolve target
-    const descriptor: GPURenderPassDescriptor = {
-      colorAttachments: [t.getColorAttachment(multisampledColorTexture)],
-    };
-    t.tryRenderPass(true, descriptor);
-  }
-  {
-    // It is not allowed to use multiple color attachments with different sample counts
-    const descriptor: GPURenderPassDescriptor = {
-      colorAttachments: [
-        t.getColorAttachment(colorTexture),
-        t.getColorAttachment(multisampledColorTexture),
-      ],
-    };
+    {
+      // It is allowed to use a multisampled color attachment without setting resolve target
+      const descriptor: GPURenderPassDescriptor = {
+        colorAttachments: [t.getColorAttachment(multisampledColorTexture)],
+      };
+      t.tryRenderPass(true, descriptor);
+    }
+    {
+      // It is not allowed to use multiple color attachments with different sample counts
+      const descriptor: GPURenderPassDescriptor = {
+        colorAttachments: [
+          t.getColorAttachment(colorTexture),
+          t.getColorAttachment(multisampledColorTexture),
+        ],
+      };
 
-    t.tryRenderPass(false, descriptor);
-  }
-});
+      t.tryRenderPass(false, descriptor);
+    }
+  });
 
-g.test('it_is_invalid_to_use_a_multisampled_resolve_target').fn(async t => {
-  const multisampledColorTexture = t.createTexture({ sampleCount: 4 });
-  const multisampledResolveTargetTexture = t.createTexture({ sampleCount: 4 });
+g.test('color_attachments_multisampled_resolve_target')
```

Comment:
perhaps `resolveTarget,sample_count` (this tests the sample count of the resolveTarget member)

The other resolveTarget tests should also mention it by name.

Reply:
ok, let me apply this name pattern to the resolveTarget tests in the following PR.